articles/connections/enterprise/saml.md

```diff
@@ -187,6 +187,22 @@ Use the `metadataUrl` option to provide the URL of the document:
 
 When providing the URL, content is downloaded only once; the connection will not automatically reconfigure if the content of the URL changes in the future.
 
+## Specify a custom Entity ID
+By default, the connection will expect the SAML IdP to return an Entity ID (or Audience) in the SAML assertion as `urn:auth0:YOUR_TENANT:YOUR_CONNECTION_NAME.` In some use cases, you might need to set your own custom Entity ID value using the Management API. You can set the `connection.options.entityID` property when the connection is first created or by updating an existing connection.
+
+The JSON example below can be used to create a new SAML connection using the SAML IdP’s metadata URL while also specifying a custom Entity ID. The Entity ID is still unique as it is created using the name of the connection.
```

Comment:
Recommend removing passive voice: "You can use the JSON example below to create a new SAML connection that uses the SAML IdP's metadata URL and specifies a custom Entity ID. The Entity ID is unique because it uses the name of the connection.